apps/docs/components/Navigation/Footer.tsx

```diff
@@ -5,11 +5,98 @@ import { LayoutMainContent } from '~/layouts/DefaultLayout'
 
 const Footer = () => (
   <LayoutMainContent className="pt-0">
-    <div className="mt-16">
-      <ul className="flex flex-col gap-2">
-        {primaryLinks.map(({ url, featherIcon: Icon, icon, text, ctaLabel }) => (
-          <li key={url} className="flex items-center gap-1 text-xs text-foreground-lighter">
-            {icon && (
+    <footer role="contentinfo" aria-label="footer">
```

Comment:
most of this diff is just indentation changes, suggest turning on "Hide whitespace" to make this easier to review